NFe.Servicos/ServicosNFe.cs

```diff
@@ -79,17 +79,22 @@ namespace NFe.Servicos
     public sealed class ServicosNFe : IDisposable
     {
         private readonly X509Certificate2 _certificado;
+        private readonly bool _controlarCertificado;
```

Comment:
Variável para saber se o Zeus deve controlar o certificado ou não

Reply:
@marcosgerene , o que exatamente faz este controle de certificado do Zeus?

Reply:
@apbertoletti se a variável for true quem vai chamar o "ObterCertificado()" e posteriormente o reset dele será o Zeus, caso contrário o desenvolvedor pode controlar isso externamente